react/AppIcon/index.jsx

```diff
@@ -7,16 +7,129 @@ import Icon from '../Icon'
 
 import appDefaultIcon from '../../assets/icons/ui/cube.svg'
 
+let domain
+const _getDomain = () => {
+  if (domain) return domain
+  const root = document.querySelector('[role=application]')
+  if (!root) {
+    throw new Error('Cannot find [role=application] element')
+  }
+  domain = root.dataset.cozyDomain
+  if (!domain) {
+    throw new Error(
+      'Application element does not contain `data-cozy-domain` attribute'
+    )
+  }
+  return domain
+}
```

Comment:
After a mobile logout, this variable will not be cleared and thus domain will have the wrong value after a second login.

Reply:
The only solution I see is to always perform the `document.querySelector`. Do you see any other option ?

Reply:
This won't work in mobile as we do not have the cozyDomain on the DOM node. To me, AppIcon should work in the context of a client. And the domain should be taken from the client.